Add mention of Arrow port to AGA docs page (#1072)
neo4j/docs-aura: modules/ROOT/pages/graph-analytics/aga.adoc

* The maximum available memory for each session
* The maximum number of concurrent sessions within the organization

=== Network requirements

> Aura Graph Analytics sessions communicate with your AuraDB instance (or external data source) using the Apache Arrow Flight protocol for remote projections and write-back.

Reviewer: i would change the phrasing a bit. The reason you need to enable the Arrow port is so the user can connect to the AGA session. The AuraDB -- AGA session is internal only and has no config impact for the user.

```suggestion
Aura Graph Analytics sessions use the Apache Arrow Flight protocol for communication from and to your client.
```

mainly its only important if you are interacting with AGA from your client machine and not directly inside Aura

Author: [raised a separate PR](https://github.com/neo4j/docs-aura/pull/1076)
